cmd/clusterctl/clusterdeployer/bootstrap/kind/kind.go

```diff
@@ -55,7 +57,7 @@ func WithOptions(options []string) *Kind {
 		}
 		return true
 	}() {
-		options = append(options, fmt.Sprintf("name=%s", kindClusterName))
+		options = append(options, fmt.Sprintf("name=%s", kindClusterNamePrefix+util.RandomString(5)))
```

Comment:
Wouldn't this break create/delete given that we generate a new name every time instead of a static one?

Reply:
It *should* be fine since a different cluster should be used for `clusterctl create` vs `clusterctl delete` and for a given `clusterctl` action the name is generated on instantiation of the bootstrapper.